src/Uno.UWP/Globalization/NumberFormatting/Rounder.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+
+namespace Windows.Globalization.NumberFormatting
+{
+    public static class Rounder
```

Comment:
Please make this class `internal` as it is not part of the public UWP/WinUI contract. You can also move it to `Uno.Globalization.NumberFormatting` namespace

Reply:
@MartinZikmund Is the file location correct? Where should I put the file?